<?xml version="1.0" ?>
<sdf version="1.5">
  <model name="charuco_board">
    <static>true</static>
    <link name="link">
      <collision name="collision">
        <geometry>
          <box>
            <size>0.08 0.8 0.8</size>
          </box>
        </geometry>
        <surface>
          <friction>
            <ode>
              <mu>100</mu>
              <mu2>50</mu2>
            </ode>
          </friction>
        </surface>
      </collision>

      <visual name="visual">
        <geometry>
          <box>
            <size>0.08 0.8 0.8</size>
          </box>
        </geometry>
        <material>
          <script>
            <uri>model://charuco_board/materials/scripts</uri>
            <uri>model://charuco_board/materials/textures</uri>
            <name>charuco_board/Diffuse</name>
          </script>
        </material>
      </visual>
    </link>
  </model>
</sdf>
